src/Components/Header.jsx

```diff
@@ -4,7 +4,7 @@ const Header = () => {
   return (
     <header>
       <h1 className="header">
-        <u>NS Marketplace</u>
+        <u>NS News</u>
```

Comment:
I wouldnt use the U tag - if you want it underlined, you can do that in the css/text-decoration